user/api.go

```diff
@@ -456,7 +458,7 @@ func (a *Api) CreateUser(res http.ResponseWriter, req *http.Request) {
 // @Failure 400 {object} status.Status "message returned:\"Invalid user details were given\" "
 // @Router /user/{userid} [put]
 func (a *Api) UpdateUser(res http.ResponseWriter, req *http.Request, vars map[string]string) {
-	a.logger.Printf("UpdateUser %v", req)
+	a.logger.WithField("trace_token", sanitizeSessionTrace(req)).Infof("processing a update user request")
```

Comment:
Perhaps usefull to add a trace of the decoded payload (as in createUser, see above) ?